runtime/caml/domain.h

```diff
@@ -70,6 +70,7 @@ void caml_reset_young_limit(caml_domain_state *);
 
 CAMLextern void caml_reset_domain_lock(void);
 CAMLextern int caml_bt_is_in_blocking_section(void);
+CAMLexport int caml_bt_is_self(void);
```

Comment:
This fixes some of the Windows failures, but not all:
```suggestion
CAMLextern int caml_bt_is_self(void);
```

Reply:
Ah, to expand - the Windows workers in Jenkins are running binutils 2.34, so don't surface the bug which this fixes. The other failure I'm seeing locally is in the `tests/output-complete-obj/main.ml`, but that doesn't appear to be this PR

Reply:
(indeed - the failure with `tests/output-complete-obj/main.ml` seems to be caused by binutils 2.38, which I updated to two days ago <sigh>)

Reply:
Thanks for catching this. This is fixed now.